src/main/scala/com/github/sbt/git/ConsoleGitRunner.scala

```diff
@@ -17,15 +19,15 @@ object ConsoleGitRunner extends GitRunner {
 
   // in order to enable colors we trick git into thinking we're a pager, because it already knows we're not a tty
   val colorSupport: Seq[(String, String)] =
-    if(ConsoleLogger.formatEnabled) Seq("GIT_PAGER_IN_USE" -> "1")
+    if(ConsoleAppender.formatEnabledInEnv) Seq("GIT_PAGER_IN_USE" -> "1")
```

Comment:
Both methods are deprecated, the latter points to `Terminal.isColorEnabled`, but that one's private. I'm not sure if these will be removed in the final 2.x, but for now I couldn't find a suitable replacement.